app/controllers/carts_controller.rb

```diff
@@ -6,12 +6,13 @@ def show
 
   def update
     product = Product.find(params[:product_id])
-    @cart.update_item_quantity(product.id, params[:quantity])
+    @cart.update_item_quantity(product.id, params[:quantity].to_i)
   end
 
   private
 
   def set_cart
     @cart = Cart.find_or_create_by(session_key: session.id.to_s)
+    Current.cart = @cart
```

Comment:
After introducing `Current.cart` do we still need to have an instance variable `@cart` or it can be replaced with a local var?